ui/app/src/components/workspaces/WorkspaceProvider.tsx

```diff
@@ -105,47 +135,67 @@ export const WorkspaceProvider: React.FunctionComponent = () => {
-                addWorkspaceService={(ws: WorkspaceService) => addWorkspaceService(ws)} />
-            </Stack.Item><Stack.Item className='tre-body-content'>
+            {!refIsTREAdminUser.current && (
+              <Stack.Item className='tre-left-nav'>
+                <WorkspaceLeftNav
+                  workspaceServices={workspaceServices}
+                  sharedServices={sharedServices}
+                  setWorkspaceService={(ws: WorkspaceService) => setSelectedWorkspaceService(ws)}
+                  addWorkspaceService={(ws: WorkspaceService) => addWorkspaceService(ws)} />
+              </Stack.Item>
+            )}
+            <Stack.Item className='tre-body-content'>
               <Stack>
                 <Stack.Item grow={100}>
                   <Routes>
                     <Route path="/" element={<>
                       <WorkspaceItem />
-                      <WorkspaceServices workspaceServices={workspaceServices}
-                        setWorkspaceService={(ws: WorkspaceService) => setSelectedWorkspaceService(ws)}
-                        addWorkspaceService={(ws: WorkspaceService) => addWorkspaceService(ws)}
-                        updateWorkspaceService={(ws: WorkspaceService) => updateWorkspaceService(ws)}
-                        removeWorkspaceService={(ws: WorkspaceService) => removeWorkspaceService(ws)} />
-                    </>} />
-                    <Route path="workspace-services" element={
-                      <WorkspaceServices workspaceServices={workspaceServices}
-                        setWorkspaceService={(ws: WorkspaceService) => setSelectedWorkspaceService(ws)}
-                        addWorkspaceService={(ws: WorkspaceService) => addWorkspaceService(ws)}
-                        updateWorkspaceService={(ws: WorkspaceService) => updateWorkspaceService(ws)}
-                        removeWorkspaceService={(ws: WorkspaceService) => removeWorkspaceService(ws)}
-                      />
-                    } />
-                    <Route path="workspace-services/:workspaceServiceId/*" element={
-                      <WorkspaceServiceItem
-                        workspaceService={selectedWorkspaceService}
-                        updateWorkspaceService={(ws: WorkspaceService) => updateWorkspaceService(ws)}
-                        removeWorkspaceService={(ws: WorkspaceService) => removeWorkspaceService(ws)} />
-                    } />
-                    <Route path="shared-services" element={
-                      <SharedServices readonly={true} />
-                    } />
-                    <Route path="shared-services/:sharedServiceId/*" element={
-                      <SharedServiceItem readonly={true} />
-                    } />
-                    <Route path="requests/*" element={
-                      <Airlock/>
-                    } />
+                      {!refIsTREAdminUser.current ? (
```

Comment:
`nit` I often find its cleaner with conditional ui blocks to have a separate element variable declaration from the main render body. I.e.

```tsx
const leftNav = refIsTREAdminUser.current ? <>
  <AdminLeftNavStuff/>
</> : <>
  <WorkspaceUserLeftNavStuff/>
</>
```

Then call it within the render block:

```tsx
<WorkspaceItem />
{{ leftNav }}
```

Reply:
Good to know, not sure which is best, I ain't got a lot of time, so going to leave it as is for now.